diff --git a/dist/index.mjs b/dist/index.mjs
index 7960fa7137696d6a6751359bb634aa215a18a3e9..f21c9b4a2fbf4e6ca02c54b9e81d098ba4ae1431 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -103,7 +103,7 @@ async function generateCSS(labels, options) {
   border-top-right-radius: 0 !important;
 }
 `;
-  const mergedIcons = { ...options.customIcon, ...builtinIcons };
+  const mergedIcons = { ...builtinIcons, ...options.customIcon };
   const matched = getMatchedLabels(labels, mergedIcons);
   const css = baseCSS + await generateIconCSS(matched);
   return { css };
